src/BackupCode/RegisterHandler.php

```diff
@@ -103,7 +136,7 @@ public function getDescription()
      */
     public function getSupportLink()
     {
-        return ''; // TODO Will this have a user help link?
+        return static::config()->get('user_help_link') ?: '';
```

Comment:
```suggestion
        return (string) $this->config()->get('user_help_link');
```

I don't see any value in using `static::config()`. It's also a bit confusing with LSB when you think about classes extending this. I'd expect people to change this via YAML on this particular class.

Reply:
Can I change it to `self::config()` as the trait defines the method as `static`?

Reply:
Reminds me of the same argument from somebody with assertions in PHPUnit, where the library author replied saying "no, use `$this->assert...`". Up to you, this was a minor suggestion because I think non-static access reads nicer when you don't _have_ to use static access

Reply:
Ok I'll use `$this->config()` from now on.